ffi/src/lib.rs

```diff
@@ -13,13 +13,20 @@
     )
 )]
 
+#[cfg(feature = "logger")]
+use env_logger::Target::Pipe;
+#[cfg(feature = "logger")]
+use std::fs::OpenOptions;
+#[cfg(feature = "logger")]
+use std::path::Path;
```

Comment:
nit/tip, since these are only used within the `start_logs` where the feature is enabled, you can move the imports to inside the function (at the top). This way you can remove the cfg attributes and keep the conditional imports closer to where they are used.